src/agent/rustjail/src/mount.rs

```diff
@@ -97,8 +97,16 @@ lazy_static! {
         m.insert("rbind", (false, MsFlags::MS_BIND | MsFlags::MS_REC));
         m.insert("relatime", (false, MsFlags::MS_RELATIME));
         m.insert("norelatime", (true, MsFlags::MS_RELATIME));
+        m.insert("silent", (false, MsFlags::MS_SILENT));
+        m.insert("loud", (true, MsFlags::MS_SILENT));
+        m.insert("acl", (false, MsFlags::MS_POSIXACL));
+        m.insert("noacl", (true, MsFlags::MS_POSIXACL));
+        m.insert("iversion", (false, MsFlags::MS_I_VERSION));
+        m.insert("noiversion", (true, MsFlags::MS_I_VERSION));
         m.insert("strictatime", (false, MsFlags::MS_STRICTATIME));
         m.insert("nostrictatime", (true, MsFlags::MS_STRICTATIME));
+        m.insert("lazytime", (false, MsFlags::MS_LAZYTIME));
+        m.insert("nolazytime", (true, MsFlags::MS_LAZYTIME));
```

Comment:
Not strictly required for this PR, but you might like to add a second commit to sort `PROPAGATION` and `OPTIONS` alphabetically. Even though a number of these entries are effectively in pairs ("`foo`, "`nofoo`"), it's hard to visually scan over the list currently.

Reply:
Thank you for the comment.
You're completely right .
I created the second commit and added the description to the current issue #1999.

Reply:
Thanks very much @ManaSugi !